Makefile

```diff
@@ -50,8 +51,8 @@ all: Makefile .docker-image conf
 	$(call docker_run, make bb/${IMAGE_RECIPE})
 
 .PHONY: bash
-bash: .docker-image
-	$(call docker_run, /bin/bash)
+bash:
```

Comment:
This looks like it might be an accidental change.  Why remove the dependency on .docker-image?

Reply:
That's a rebase mistake.

Reply:
The most recent force push should have fixed this mistake.